libs/lein-template/resources/leiningen/new/kit/deps.edn

```diff
@@ -53,9 +53,11 @@
                                 pjstadig/humane-test-output     {:mvn/version "0.11.0"}
                                 ring/ring-devel                 {:mvn/version "1.9.4"}
                                 ring/ring-mock                  {:mvn/version "0.4.0"}
+                                nrepl/nrepl                     {:mvn/version "0.9.0"}
+                                cider/cider-nrepl               {:mvn/version "0.28.0"}
```

Comment:
a hardcoded  cider nrepl version may broke emacs cider plugin. 
it's redundant to  define cider with version here

Reply:
@sound2gd  that's true, since `cider-nrepl` version needs to match the version of Cider installed with Emacs.

The issue is that if `cider-nrepl` is not defined here, connecting to nrepl from Cider using `cider-connect` gives a warning about missing library. That's why I included here cider-nrepl that matches the latest Cider from melpa.

I am not aware of a user-friendly way to have dynamic version here, except for editing deps.edn after project creation to match the actual Cider version.

Reply:
I just use cider-jack-in with additional dependencies, the docs could be found [here](https://docs.cider.mx/cider/basics/up_and_running.html)